.github/CODEOWNERS

```
@@ -67,6 +67,9 @@
 /nixos/lib/make-disk-image.nix                 @raitobezarius
 
 # Nix, the package manager
+# @raitobezarius is not "code owner", but is listed here to be notified of changes
+# pertaining to the Nix package manager.
+# i.e. no authority over those files.
 pkgs/tools/package-management/nix/                    @raitobezarius @ma27
 nixos/modules/installer/tools/nix-fallback-paths.nix  @raitobezarius @ma27
```

I think this only creates confusion. Since the github UI, will state that you are the code owner after all, even you are not and no one checks this file.

```suggestion
pkgs/tools/package-management/nix/                    @ma27
nixos/modules/installer/tools/nix-fallback-paths.nix  @ma27
```

---

Hm, can you elaborate on how email filter can provide a way to follow changes? (genuinely don't see how to do it.)

---

Something like `subject =~ /^(nix|nixVersions.*):/`. But anyway. This is not the hill I would die on. Do you as you wish. I just find the outcome suboptimal.

---

Well, I am all in favor of a better solution but I don't want to find myself engineering a solution _right now_, so this outcome may be suboptimal but I am also very willing to change it later.